game/src/js/scenes/CharacterSelectionScene.js

```diff
@@ -60,6 +60,17 @@ export default class CharacterSelectionScene extends Phaser.Scene {
     chkButton.on('pointerover', () => {
       this.showCharacter(btnData.key);
     });
+    
+    let backToMenuButton = new Button(this, 450, 610, 'Return To Menu!');
+    backToMenuButton.props={
```

Comment:
These custom settings should be passed into the constructor. Right now this is just overwriting the props with a subset of the necessary variables, which is likely to cause problems.